ui-v2/app/components/list-collection/index.hbs

```diff
@@ -1,4 +1,4 @@
-<EmberNativeScrollable @tagName="ul" @content-size={{_contentSize}} @scroll-left={{_scrollLeft}} @scroll-top={{_scrollTop}} @scrollChange={{action "scrollChange"}} @clientSizeChange={{action "clientSizeChange"}}>
+<EmberNativeScrollable class={{class}} @tagName="ul" @content-size={{_contentSize}} @scroll-left={{_scrollLeft}} @scroll-top={{_scrollTop}} @scrollChange={{action "scrollChange"}} @clientSizeChange={{action "clientSizeChange"}}>
```

Comment:
I'm not totally sure, but is this `class={{class}}` needed at all? Pretty sure its not needed here, but I might be wrong.

Reply:
Removed class